### DISCUSSION TITLE
What is opengl_default?

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#3743] [Q&A] [answered]
What's the difference between `output=opengl_default` and `output=opengl`? Only the second is described in .conf.

![image](https://github.com/dosbox-staging/dosbox-staging/assets/97228894/6c138a5d-0395-43b6-9682-add8e63f04db)

edit: PR fix #3753

**Comment by johnnovak** [comment]
It's a bug introduced by @japsmits changes I believe. It shouldn't be there.

@japsmits would you be able to take a look?

**Reply by japsmits** [reply, marked as answer]
I found the culprit and will create an issue to fix it.

Thanks for taking the time to report this! 👍
